definitions/variable/emissions/tag_ocean-removal-option.yaml

```diff
@@ -0,0 +1,7 @@
+- Ocean Removal Option:
+    - Ocean Alkalinity Enhancement:
+        description: increasing ocean alkalinity
```

Comment:
"increasing ocean uptake of carbon through anthropogenic alkalinity addition"

Reply:
Done.